src/lib/Button/Button.svelte

```
@@ -22,16 +32,23 @@
-    showProgressBar = $bindable(false),
     icon,
     children,
     classes
   }: ButtonProperties = $props();
 
-  let isDisabled = $derived(!enable || disabled || showLoader);
+  let isLoading = $derived(loading || showLoader);
+  let isDisabled = $derived(disabled || !enable || isLoading);
+  // `loading` and the legacy Circular loaderType both render the spinner.
+  let showCircularLoader = $derived(loading || (showLoader && loaderType === 'Circular'));
+  let resolvedRel = $derived(rel ?? (target === '_blank' ? 'noopener noreferrer' : undefined));
```

ESLint

[error] undefined is not allowed. Use null or proper type checking instead.

(no-restricted-syntax)